src/app/cdk/panel/panels/panels.component.ts

```diff
@@ -181,6 +185,9 @@ export class PanelsComponent implements OnInit {
   }
 
   ngOnInit(): void {
+    this._togglz
```

Comment:
Can we do this subscription only of `type='professionalActivitiesTogglz'` so we avoid so many subscribers? 

Also it might be a good idea to use `pipe(take(1))` when we use `getStateOf` since subscription is never completed by itself.

Reply:
makes sense I will do it